src/openrct2/peep/Guest.cpp

```diff
@@ -2272,7 +2272,7 @@ bool Guest::ShouldGoToShop(Ride* ride, bool peepAtShop)
 
         // The amount that peeps are willing to pay to use the Toilets scales with their bathroom stat.
         // It effectively has a minimum of $0.10 (due to the check above) and a maximum of $0.60.
-        if (ride->price * 40 > toilet)
+        if (!(gParkFlags & PARK_FLAGS_NO_MONEY) && ride->price * 40 > toilet)
```

Comment:
We have the `ride_get_price()` function, which checks this for you. Using it would make the code a bit cleaner.

Reply:
Argh i thought there was something like that